src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -814,6 +815,11 @@ PooledHashSet<CaptureId> adjustAndGetEntryState(ArrayBuilder<PooledHashSet<Captu
                         longLivedIds.Add(id);
                     }
 
+                    if (reference.IsInitialization)
+                    {
+                        state.Add(id);
```

Comment:
>state.Add(id); [](http://example.com/codeflow?start=24&length=14)

Is there a need to perfom checks below? It looks like we don't do that for IFlowCaptureOperation

Reply:
I don't think so.